server/src/concepts/user/controllers/index.js

```diff
@@ -97,3 +98,23 @@ export const makeUserAnAdmin = async (req, res) => {
     return res.status(500).send("User couldn't be changed to Admin");
   }
 };
+
+export const getOneUser = async (req, res) => {
+  try {
+    const user = await getUser(req.params.id);
+    if (!user) return res.status(404).send("No user found!");
+    return res.status(200).send(user);
+  } catch (error) {
+    return res.status.send(500).send(error.message);
+  }
+};
+
+export const getUsers = async (req, res) => {
+  try {
+    const allUsers = await getAllUsers();
+    if (!allUsers) return res.status(404).send("No user found!");
```

Comment:
allUsers always be true, even if there is no users because it will return empty array which is true in JS. In my opinion it is good to return empty array if you want to get all items. So this if is not neccessary.